config_template/test_param_config_template.h

```diff
@@ -50,6 +50,19 @@
  * #define MQTT_SERVER_PORT       (8883)
  */
 
+/**
+ * @brief The IoT Thing name for the device for OTA test and MQTT test.
```

Comment:
Suggest to add comment to clarify that the IOT_THING_NAME is also used as MQTT client identifier during the test.

Reply:
Added